docs/geedocs/5.2.4/answer/7160004.html

```diff
@@ -44,6 +44,8 @@ <h5>
   </h5>
     <p><strong>C++98 no longer supported</strong>. As of 5.2.4 C++11 syntax can be used and building open GEE using C++98 standard will no longer be supported</p>
     <p><strong>Added "Save and Build" file menu option</strong>. As of 5.2.4 the Fusion UI will include an additional file menu option that allows users to save and build assets with one click, instead of clicking the save option and then the build option immediately after.</p>
+    <p><strong>Added build parameter to redirect the build output</strong>Build output can now be optionally directed to a folder of your choosing instead of the current hard coded defaults. See earth_enterprise/BUILD.md in source code for more details.</p>
+    <p><strong>Added experimental build cache folder parameter to the build</strong>Currently this was added as an experimental option to potentially speed up builds. Some more work remains to be done before using this option in production builds. Use of this option is not recommend except for giving feedback and/or helping to address issues with open GEE's build process using scons cache. See earth_enterprise/BUILD.md in source code for more details.</p>
```

Comment:
Typo: `not recommend` -> `not recommended`

Reply:
For both of the new items: I think after the close of the strong tag there needs to be a period and a space.